packages/docmanager/src/savehandler.ts

```diff
@@ -38,6 +38,7 @@ class SaveHandler implements IDisposable {
     this._context = options.context;
     this._minInterval = options.saveInterval * 1000 || 120000;
     this._interval = this._minInterval;
+    this._warnOnConflict = this._context.model.modelDB.isCollaborative;
```

Comment:
Whoops, you are right. Mismatch between the name and the intent (though it currently has the right behavior). Fixing now.